database/factories/ImportFailureFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\ImportFailure;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ImportFailureFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = ImportFailure::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        $randomLine = $this->faker->optional(0.8)->numberBetween(0, 10000);
+        $randomWordAmount = $this->faker->numberBetween(1, 15);
+        $randomSentence = $this->faker->sentence($randomWordAmount);
+        $message = $randomLine ? $this->faker->randomElement([
+            __('validation.import.error', [
+                'line' => $randomLine,
+                'message' => $randomSentence
+            ]),
+            __('parsing.import.error', [
+                'line' => $randomLine,
+                'message' => $randomSentence
+            ])
+        ]) : $randomSentence;
+
```

Comment:
brilliant - I like how much love the details of random message generation get here

Reply:
Ah Schtop :blush: Yeah I find it useful to create fakers that try to mimic a situation close to reality, because it helps with the GUI later on.